### PR TITLE
CEDS-2547 Update error messages

### PR DESCRIPTION
**Pull request by MaciejRewera**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -380,8 +380,8 @@ declaration.additionalActors.remove.empty = Select yes if you need to remove thi
 declaration.additionalActors.remove.title = Are you sure you want to remove this party?
 
 
-declaration.eori.empty = Enter an EORI in the correct format. For example, GB123456789000.
-declaration.eori.error.format = Enter an EORI in the correct format. For example, GB123456789000.
+declaration.eori.empty = Enter an EORI number in the correct format. For example, GB123456789000.
```
Comment (leszeklorek) [inline conf/messages.en:383]: should it be "For example: " or this form is also correct ?

Comment (MaciejRewera) [reply conf/messages.en:383]: This is exactly how it was in the spreadsheet. I'll ask JW about it

Comment (leszeklorek) [reply conf/messages.en:383]: I am not saying it is wrong :) just being curious

Comment (MaciejRewera) [reply conf/messages.en:383]: Confirmed: it should be comma.

**Review by SteveSugden — APPROVED** [review]
LGTM

**Pull request merged**